domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/chronoLocalDateAssertions.kt

```diff
@@ -0,0 +1,15 @@
+package ch.tutteli.atrium.domain.robstoll.lib.creating
+
+import ch.tutteli.atrium.assertions.Assertion
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.translations.DescriptionDateTimeLikeAssertion
+
+fun <T: ChronoLocalDate> _isBeforeOrEquals(assertionContainer: Expect<T>, expected: T):Assertion =
+    ExpectImpl
+        .builder
+        .createDescriptive(assertionContainer, DescriptionDateTimeLikeAssertion.IS_AFTER, expected)
```

Comment:
please import DescriptionDateTimeLikeAssertion.IS_AFTER, this way the body of the function fits on one line